modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/controller/ExportController.scala

```diff
@@ -46,4 +52,62 @@ class ExportController {
       throw new ForbiddenException()
     }
   }
+
+  @PostMapping(value = Array("/tasks/export"), produces = Array(MediaType.APPLICATION_JSON_VALUE))
```

Comment:
```suggestion
  @PostMapping(value = Array("/tasks/export"))
```

This configuration is not necessary.